templates/registration/settings.html

```diff
@@ -3,10 +3,10 @@
 {% block content %}
     <h2>Settings</h2>
     <ul class="tabs nav-settings" data-tabs id="settings-tabs">
-        <li class="tabs-title {% if 'edit_name' in forms_with_errors or not forms_with_errors %}is-active{% endif %}"><a href="#name">Name</a></li>
-        <li class="tabs-title {% if 'edit_avatar' in forms_with_errors %}is-active{% endif %}"><a href="#avatar">Avatar</a></li>
-        <li class="tabs-title {% if 'change_password' in forms_with_errors %}is-active{% endif %}"><a href="#password">Password</a></li>
-        <li class="tabs-title {% if 'email_preferences' in forms_with_errors %}is-active{% endif %}"><a href="#email-preferences">Email Preferences</a></li>
+        <li class="tabs-title {% if 'edit_name' in forms_with_errors or not forms_with_errors %}is-active{% endif %}"><a href="#name"><i class="fa fa-user fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Name</span></a></li>
+        <li class="tabs-title {% if 'edit_avatar' in forms_with_errors %}"><a href="#avatar"><i class="fa fa-photo fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Avatar</span></a></li>
```

Comment:
It looks like you erased `is-active{% endif %}` from this line. Removing that breaks this page.